ios/sdk/src/JitsiMeet.m

```diff
@@ -53,10 +57,13 @@ + (instancetype)sharedInstance {
 
 - (instancetype)init {
     if (self = [super init]) {
+        
         // Initialize WebRTC options.
         self.rtcAudioDevice = nil;
         self.webRtcLoggingSeverity = WebRTCLoggingSeverityNone;
 
+        [self createReactNativeFactory];
```

Comment:
We cannot call this so early. The bridge / factory should exist for the first time when we join a meeting.